### PR TITLE
Improve the readme

### PR DESCRIPTION
**Pull request by ToddBradley**
1. Make the commands be all-poetry instead of a mix of pip3/venv and poetry
2. Make the readme Markdown a little easier to read

### CODE REVIEW

File: python_examples/remove_uuid_tags/README.md [hunk 1]
```diff
@@ -1,52 +1,37 @@
...
-*** exporting an authentication token for use in the virtual environment ***
+## Exporting an authentication token for use in the virtual environment
+
 - In your portfolio click in the top right corner and under settings select Manage Access Tokens 
 - Once you generate a token make sure that you view the secret it should look something like this 
-  API-V1 xxxxxxxxxxxxxxxxxxxxxx==.xxxxxxx+xxxxxxxxxxxxxxxxxxxxx/xxxxxxxxxxxxxx="
-- using a text editor of your choice create a .auth-token file
-- Inside the .auth-token file take your token secret and add:
-  export CYBERGRX_API_TOKEN="API-V1 xxxxxxxxxxxxxxxxxxxxxx==.xxxxxxx+xxxxxxxxxxxxxxxxxxxxx/xxxxxxxxxxxxxx="
+  `API-V1 xxxxxxxxxxxxxxxxxxxxxx==.xxxxxxx+xxxxxxxxxxxxxxxxxxxxx/xxxxxxxxxxxxxx="`
```
Comment (brian-l) [inline python_examples/remove_uuid_tags/README.md]: I think the double quote shouldn't be there

```suggestion
  `API-V1 xxxxxxxxxxxxxxxxxxxxxx==.xxxxxxx+xxxxxxxxxxxxxxxxxxxxx/xxxxxxxxxxxxxx=`
```

File: python_examples/remove_uuid_tags/README.md [hunk 2]
```diff
@@ -1,52 +1,37 @@
...
+- using a text editor of your choice create a `.auth-token` file
```
Comment (brian-l) [inline python_examples/remove_uuid_tags/README.md]:
```suggestion
- Using a text editor of your choice create a `.auth-token` file
```

**Review by brian-l — CHANGES_REQUESTED** [review]

File: python_examples/remove_uuid_tags/README.md (hunk 1 above)
Comment (ToddBradley) [reply python_examples/remove_uuid_tags/README.md]: I think you're right. @jsully1720 did you test that these instructions work with the double quotes? And do they work *without* the double quotes?

Comment (jsully1720) [reply python_examples/remove_uuid_tags/README.md]: That's spot on, that's my fault. It definitely wouldn't work like that

Comment (ToddBradley) [reply python_examples/remove_uuid_tags/README.md]: (note that if we remove the ending double quote on line 15, we also need to remove the starting double quote on the line above - whether they are needed or not, they need to match)

**Review by brian-l — APPROVED** [review]

File: python_examples/remove_uuid_tags/README.md [hunk 3]
```diff
@@ -1,52 +1,37 @@
...
-7. deactivate
 
-* remove .auth-token
-8. rm -rf .auth-token
+```
+poetry install
+source .auth-token
+poetry run python3 removeTags.py 
+rm -rf .auth-token
+```
```
Comment (jsully1720) [inline python_examples/remove_uuid_tags/README.md:36]: The problem with completely removing the venv is we don't know who the customer using this is. From my understanding internally we create the venv and then tell poetry to use it. From a least likelihood of breaking standpoint creating a venv env and then having poetry install on top of that seems like that best chance of success?

**Review by jsully1720 — APPROVED** [review]

**Pull request merged**